hwmgr-plugins/api/middleware.go

```diff
@@ -0,0 +1,120 @@
+/*
+SPDX-FileCopyrightText: Red Hat
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
```

Comment:
We do have a middleware.go [here](https://github.com/openshift-kni/oran-o2ims/blob/9c32b3104b300b2e194fbdda899580e6a869f8d5/internal/service/common/api/middleware/middleware.go#L1). Wondering why there's a copy instead? There are a few things you might need later e.g allowing trailing slash, improved uuid validation etc.

Reply:
There will be an opportunity to refactor and cleanup the `hwmgr-plugins` content once we have fully restructured the hardware manager and plugins.